stackcopy_gui.py

```
@@ -0,0 +1,481 @@
+    # -- cancel / open / close --------------------------------------------
+
+    def _on_cancel(self) -> None:
+        proc = self._proc
+        if proc and proc.poll() is None:
+            # Each file move is atomic and the import is re-runnable, so stopping
+            # between files is safe.
+            self.status_var.set("Cancelling...")
+            try:
+                proc.terminate()
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)